tests/phpunit/integration/Modules/Search_ConsoleTest.php

```diff
@@ -192,6 +195,24 @@ public function test_get_module_scopes() {
 		);
 	}
 
+	public function test_data_available_reset_on_property_change() {
+		$search_console = new Search_Console( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
+		$search_console->register();
+		$search_console->get_settings()->merge(
+			array(
+				'propertyID' => 'https://example.com',
+			)
+		);
+		$search_console->set_data_available();
+		$search_console->get_settings()->merge(
+			array(
+				'propertyID' => 'https://example2.com',
```

Comment:
Nit: better to keep the domain the same and add a base path to avoid using a potential real domain.
```suggestion
				'propertyID' => 'https://example.com/other/',
```

Reply:
Good point. There were a few more instances of `example2.com` and `example1.com`. I have changed `example1.com` to `example.com` and `example2.com` to `example.org` since that's also a reserved domain that we've used in other places and is easier to reason about at a glance. What do you think?

Reply:
Sounds good to me – the WP PHPUnit tests default to use `example.org` as the site's domain as well. https://github.com/WordPress/wordpress-develop/blob/b55f01db1dd0c8de654e4175121337fe25f24dca/wp-tests-config-sample.php#L64